CHANGELOG.asciidoc

```diff
@@ -3,6 +3,238 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.12.0]]
+=== Beats version 7.12.0
+https://github.com/elastic/beats/compare/v7.11.2...v7.12.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Fix panic with inline SSL when the certificate or key were small than 256 bytes. {pull}23820[23820]
+
+*Auditbeat*
+
+*Filebeat*
+
+- Rename `s3` input to `aws-s3` input. {pull}23469[23469]
+
+*Heartbeat*
+- Refactor synthetics configuration to new syntax. {pull}23467[23467]
+
+*Journalbeat*
+
+
+
+*Metricbeat*
+
+- Add container.image.name and containe.name ECS fields for state_container. {pull}23802[23802]
+- Add support for the MemoryPressure, DiskPressure, OutOfDisk and PIDPressure status conditions in state_node. {pull}[23905]
+
```

Comment:
@ChrsMark are these  breaking changes or shall we move them to fix?

Reply:
nope, just additions sorry for this